### ISSUE TITLE
STORY: As a team member demoing Monarch IMAGE experiences, I want to be able to switch between demo modes, so that I can show off current functionality as well as older demos without having to install separate apps.

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-Monarch#54]
Right now, the IMAGE Monarch client app largely one big experience in a core activity. Older demos, such as the maps and photos experiences from 2023, will no longer be able to run once the new authoring environment work is merged to main. The plan is:

- [x] tag current main so that we can get back to it cleanly if we need to run older demos anytime soon
- [x] merge the authoring environment link code into main, breaking the old experiences
- [x] design way to refactor client core activity to modularize the different experiences into their own activities as much as possible (review by @JRegimbal)
- [ ] implement the refactoring
- [ ] implement a switching UI via Android controls in the Monarch app ~that will have to be manipulated on an attached monitor~ with TTS read out so that it can be done without a monitor(only team members will be doing this while demoing!)

Then, moving forward, we can use the same modular approach to implement additional types of experiences. Also makes it easy for an intern or student team to create new experiences for the Monarch without having to touch core code.

Note that the default experience when the Monarch client app is launched should be directly related to the current SOW goals.

Future work item, when designing app to incorporate KeySoft + be standalone for end users: figure out how to expose all of the functionality in the different modules in some sane way.
